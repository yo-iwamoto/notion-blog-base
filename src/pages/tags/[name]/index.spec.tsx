import { render } from '@/__tests__/util';
import Page from './index.page';

describe('[page] /tags', () => {
  it('renders /tags unchanged', async () => {
    const { container } = render(
      <Page
        name='tag name'
        pages={[
          {
            id: 'id',
            properties: {
              title: 'title',
              slug: 'slug',
              created_at: '2020-03-17T21:49:37.913Z',
              tags: [{ id: 'id', color: 'blue', name: 'React' }],
              status: 'public',
            },
          },
        ]}
      />
    );

    expect(container).toMatchSnapshot();
  });
});
