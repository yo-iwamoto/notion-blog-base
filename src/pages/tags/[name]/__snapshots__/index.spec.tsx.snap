// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[page] /tags renders /tags unchanged 1`] = `
<div>
  <div
    class="font-sans text-gray-900"
  >
    <div
      class="mx-4 mt-12"
    >
      <div
        class="mx-auto w-full max-w-2xl"
      >
        <main>
          <h1
            class="mb-12 text-4xl font-bold"
          >
            <span
              class="before:content-[\\"#\\"]"
            >
              tag name
            </span>
          </h1>
          <ul>
            <li>
              <a
                href="/posts/slug"
              >
                title
              </a>
            </li>
          </ul>
        </main>
      </div>
    </div>
  </div>
</div>
`;
